src/game/BattleGround/BattleGroundWS.h

```diff
@@ -20,11 +20,19 @@
 #define __BATTLEGROUNDWS_H
 
 #include "BattleGround.h"
+#include "Tools/Language.h"
```

Comment:
This can be moved to the source as well (see remark below about the block of statics). Fewer dependencies in headers is always good :wink: